packages/card/src/Card.tsx

```diff
@@ -1,28 +1,55 @@
 import React from 'react';
+import { transparentize } from 'polished';
 import PropTypes from 'prop-types';
 import { css, cx } from '@leafygreen-ui/emotion';
 import { uiColors } from '@leafygreen-ui/palette';
 import Box, { ExtendableBox } from '@leafygreen-ui/box';
 
+const baseBoxShadow = `0 4px 10px -4px ${transparentize(0.7, uiColors.black)}`;
+const hoverBoxShadow = `0 2px 6px -2px ${transparentize(0.4, uiColors.black)}`;
+const focusBoxShadow = '0 0 0 3px #9dd0e7';
+
 const containerStyle = css`
+  position: relative;
   background-color: white;
   border: 1px solid ${uiColors.gray.light2};
-  border-radius: 4px;
-  box-shadow: 0px 4px 10px -4px ${uiColors.gray.light1};
+  border-radius: 7px;
+  box-shadow: ${baseBoxShadow};
   transition: border 300ms ease-in-out, box-shadow 300ms ease-in-out;
+`;
+
+const clickableStyle = css`
+  cursor: pointer;
+
+  &:focus {
+    outline: none;
+    box-shadow: ${baseBoxShadow}, ${focusBoxShadow};
+  }
 
   &:hover {
     border: 1px solid ${uiColors.gray.light2};
-    box-shadow: 0px 3px 6px -2px ${uiColors.gray.base};
+    box-shadow: ${hoverBoxShadow};
+
+    &:focus {
+      box-shadow: ${hoverBoxShadow}, ${focusBoxShadow};
+    }
   }
 `;
 
 interface CardProps {
   className?: string;
+  onClick?: React.MouseEventHandler;
 }
 
 const Card: ExtendableBox<CardProps> = ({ className, ...rest }: CardProps) => {
-  return <Box className={cx(containerStyle, className)} {...rest} />;
+  const clickable = rest.onClick !== undefined;
+  return (
+    <Box
+      className={cx(containerStyle, { [clickableStyle]: clickable }, className)}
+      tabIndex={clickable ? 0 : undefined}
```

Comment:
We shouldn't need to set tabIndex on elements that would typically need an onClick handler like links and buttons. Anything else, we may want to let the consumer add tabIndex themselves in cases where they're setting an onClick handler on a typically non-interactive element.